client/src/components/BobaCard.jsx

```diff
@@ -0,0 +1,106 @@
+import { React } from "react";
+
+import { Box, Stack, Typography, Divider, useTheme, Link } from "@mui/material";
+
+import BobaCardPath from "../assets/card-milktea_p1.svg";
+
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
+import LanguageIcon from "@mui/icons-material/Language";
+
+const BobaCard = ({
+  name,
+  pronouns,
+  position,
+  linkedInUrl,
+  websiteUrl,
+  icon,
+}) => {
+  const theme = useTheme();
+
+  return (
+    <Box
+      sx={{
+        width: "90%",
+        position: "relative",
+        marginTop: "30px",
+        [theme.breakpoints.down("md")]: {
+          width: "80%",
+        },
+      }}
+    >
+      <Box component="img" src={BobaCardPath} />
+      <Stack
+        alignItems="center"
+        direction="column"
+        sx={{ position: "absolute", width: "100%", top: 0, left: 0 }}
+      >
+        <Box
+          component="img"
+          src={icon}
+          sx={{ width: "40%", marginTop: "48%" }}
+        />
+        <Typography
+          sx={{
+            marginTop: "5%",
+            fontWeight: 600,
+            textAlign: "center",
+            maxWidth: "80%",
+            fontSize: "1.3vw",
+            [theme.breakpoints.down("sm")]: {
+              fontSize: "5vw",
+            },
+          }}
+        >
+          {name}
+        </Typography>
+        <Typography
+          sx={{
+            color: "#777F8C",
```

Comment:
Is there a theme color for this instead of hard coding a color?

Reply:
the reason why a lot of the colours are hard coded is because devs discussed that it would be more convenient to go back and change it after jenna's theme reconfiguration takes place. I'll go back and replace the colours with theme colours after her ticket is merged